.env.example

```diff
@@ -215,3 +215,8 @@ PROJECT_ID_VERCEL=
 TEAM_ID_VERCEL=
 # Get it from: https://vercel.com/account/tokens
 AUTH_BEARER_TOKEN_VERCEL=
+
+#   - APPLE CALENDAR
+# Used for E2E tests on Apple Calendar
+E2E_TEST_APPLE_CALENDAR_EMAIL=""
+E2E_TEST_APPLE_CALENDAR_PASSWORD=""
```

Comment:
can we not hardcode these instead? since its just for testing? have a throway account. or is it possible to change the password?

Reply:
It is possible to change the password. Apple doesn't allow you to login into external apps using the main iCloud pass, so you must create an app-specific password.

Honestly, since the code base is public, I guess we could create a throwaway account but add this on the test pipeline rather than have it hard coded since people can try to access the account with bad intentions, and IDK how Apple protects stuff like IP of who created the account, the e-mail attached, etc.

Even though, I'm with you if you say go for the hard-coded. If so, please provide me an e-mail + app password to put in there. I tried to create one myself but had no success without attaching it to a valid e-mail/phone number.

Reply:
@keithwillcode Can we add the above to GH .env before merging this PR?